interface/resources/qml/hifi/audio/Audio.qml

```diff
@@ -677,12 +514,183 @@ Rectangle {
             }
         }
 
+        Separator {
+            id: secondSeparator;
+            anchors.top: systemInjectorGainContainer.bottom;
+            anchors.topMargin: 10;
+        }
+
+        Item {
+            id: inputDeviceHeader
+            x: margins.paddings;
+            width: parent.width - margins.paddings*2;
+            height: 36;
+            anchors.top: secondSeparator.bottom;
+            anchors.topMargin: 10;
+
+            HiFiGlyphs {
+                width: margins.sizeCheckBox;
+                text: hifi.glyphs.mic;
+                color: hifi.colors.white;
+                anchors.left: parent.left;
+                anchors.leftMargin: -size/4; //the glyph has empty space at left about 25%
+                anchors.verticalCenter: parent.verticalCenter;
+                size: 30;
+            }
+            RalewayRegular {
+                anchors.verticalCenter: parent.verticalCenter;
+                width: margins.sizeText + margins.sizeLevel;
+                anchors.left: parent.left;
+                anchors.leftMargin: margins.sizeCheckBox;
+                size: 22;
+                color: hifi.colors.white;
+                text: qsTr("Choose input device");
+            }
+        }
+        
+        AudioControls.LoopbackAudio {
+            id: loopbackAudio
+            x: margins.paddings
+            anchors.top: inputDeviceHeader.bottom;
+            anchors.topMargin: 10;
+
+            visible: (bar.currentIndex === 1 && isVR) ||
+                (bar.currentIndex === 0 && !isVR);
+            anchors { left: parent.left; leftMargin: margins.paddings }
+        }
+        
+        AudioControls.InputPeak {
+                    id: inputLevel
+                    anchors.right: parent.right
+                    peak: model.peak;
+                    anchors.verticalCenter: parent.verticalCenter
+                    visible: ((bar.currentIndex === 1 && isVR) ||
+                             (bar.currentIndex === 0 && !isVR)) &&
+                             AudioScriptingInterface.devices.input.peakValuesAvailable;
+        }
```

Comment:
This `AudioControls.InputPeal` block appears to have been moved outside of `inputView`'s `delegate: Item {` block, probably unintentionally. (The anchors don't make sense, for example.)

Reply:
Please check if the latest commit fixes the issue, as I am not 100% sure if I got what you mean.